remote/index.js

```diff
@@ -108,9 +108,7 @@ module.exports = function (app) {
       });
 
       const reportConfigFilename = path.join(
-        PROJECT_PATH,
-        'backstop_data',
-        'html_report',
+        _config.paths.html_report,
```

Comment:
@fuhlig   Sorry -- is this change deliberate?   Not sure this looks right.

Reply:
Oops, leftover. Will drop the commit

Reply:
Dropped the commit.
But while you are at it - there is a pending PR for this exact change ;)

#1172 I would appreciate if you could take a look. Thanks @garris

Reply:
Will 👀